code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -131,6 +131,7 @@ SUBSYSTEM_DEF(tickets)
 		"Already Resolved" = "The problem has been resolved already.",
 		"Mentorhelp" = "Please redirect your question to Mentorhelp, as they are better experienced with these types of questions.",
 		"Happens Again" = "Thanks, let us know if it continues to happen.",
+		"Github Issue Report" = "To report a bug, please click the 'GitHub' button at the top right of your screen, above the tabs. Then go to 'Issues'. Then 'New Issue'. Then fill out the report form. If the report would reveal current-round information, file it after the round ends.",
```

Comment:
```suggestion
		"Github Issue Report" = "To report a bug, please go to our <a href='[config.githuburl]'>Github page</a>. Then go to 'Issues'. Then 'New Issue'. Then fill out the report form. If the report would reveal current-round information, file it after the round ends.",
```

Reply:
Messed up the review formatting there, but you might want to put a link to the Github page in the response.